source/administration/changelog.md

```diff
@@ -8,6 +8,8 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
 Mattermost v5.18.0 contains low to high level security fixes. [Upgrading](http://docs.mattermost.com/administration/upgrade.html) is recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/).
 
+- **v5.18.2, released 2020-01-16**
+  - Fixed an issue with a panic in ``web.shouldSendEventToGuest``. [MM-21481](https://mattermost.atlassian.net/browse/MM-21481)
```

Comment:
What's the user (or admin) facing issue we're fixing?

Reply:
^ie. what info will an admin need to decide if they need to upgrade to 5.18.2?

Reply:
@streamer45 @agnivade Can you help describe the issue that was fixed from a user's (or admin's) point of view? For https://mattermost.atlassian.net/browse/MM-21481

Reply:
As far as I know the issue was triggered when a user update event was to be sent to a guest user in a cluster setup. So, from an admin point of view, if I had guest accounts enabled and a cluster setup I would definitely upgrade to avoid this.

@agnivade might have more info on this since he conducted the investigation and fixed the issue but he's currently OoO.

Reply:
@amyblais @streamer45 Thank you! Definitely an improvement.

Would there be an opportunity to describe the issue in simpler terms? In particular, when I read `Fixed an issue with a panic in ``web.shouldSendEventToGuest`` triggered in cases where a user update event was sent`, I don't really understand the impact. Does the app crash? Does the server crash? Does it prevent a guest from receiving notifications? Something else?

Reply:
@jasonblais 

The server crashed when the specific case occurred (`user_updated` event sent through a cluster to a guest user) and the event would not reach the guest user.

Reply:
@streamer45 has explained it perfectly. I don' have anything else to add.